src/common/Themes/default.js

```diff
@@ -0,0 +1,6 @@
+const stateColors = {
+  enabled: '#27ae60',
+  disabled: '#c0392b'
+}
+
+export default stateColors
```

Comment:
Name the file `common/themes/default.js`, and use `index-modules` to generate `common/themes/index.js` (see `package.json` and `.gitignore`).